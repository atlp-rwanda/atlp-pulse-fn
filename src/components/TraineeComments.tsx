/* eslint-disable */
import { Icon } from '@iconify/react';
import React from 'react';
import { useTranslation } from 'react-i18next';
import { IoIosArrowBack } from 'react-icons/io';
import { useNavigate } from 'react-router';
import comments from '../dummyData/comments.json';

function TraineeComments() {
  const { t } = useTranslation();
  const navigate = useNavigate();
  return (
    <div className="bg-white dark:bg-dark-bg shadow-lg px-5 py-1 rounded-md w-[90%] mx-auto lg:w-[70%] lg:ml-60 mb-10 mt-80"></div>
  );
}

export default TraineeComments;
