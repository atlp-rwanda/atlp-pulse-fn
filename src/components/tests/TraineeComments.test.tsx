/* eslint-disable */
import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import TraineeComments from '../TraineeComments';

describe('Trainee Comments page', () => {
  it('Renders the Trainee Comments Page ', () => {
    const elem = renderer
      .create(
        <MemoryRouter>
          <TraineeComments />
        </MemoryRouter>,
      )
      .toJSON();
    expect(elem).toMatchSnapshot();
  });
});
