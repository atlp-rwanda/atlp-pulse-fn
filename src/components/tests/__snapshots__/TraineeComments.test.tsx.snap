// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Trainee Comments page Renders the Trainee Comments Page  1`] = `
<div
  className="bg-white dark:bg-dark-bg shadow-lg px-5 py-1 rounded-md w-[90%] mx-auto lg:w-[70%] lg:ml-60 mb-10 mt-80"
/>
`;
